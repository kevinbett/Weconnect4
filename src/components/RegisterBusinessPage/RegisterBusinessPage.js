import React from 'react';
import RegisterBusinessForm from '../RegisterBusinessForm/index';
import Navigation from '../Navigation';

const RegisterBusinessPage = () => (

   <div>
       <Navigation />
       <RegisterBusinessForm />
       
   </div>
);

export default RegisterBusinessPage;