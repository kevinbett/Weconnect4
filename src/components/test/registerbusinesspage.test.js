import RegisterBusinessPage from '../RegisterBusinessPage/RegisterBusinessPage'
import { shallow } from "enzyme"

describe('Register business page',() => { 
    it ('Register business page renders', () => { 
        expect(shallow(<RegisterBusinessPage/>)).toMatchSnapshot()
    });
} )