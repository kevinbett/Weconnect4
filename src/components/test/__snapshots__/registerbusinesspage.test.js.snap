// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register business page Register business page renders 1`] = `
<div>
  <Navigation />
  <RegisterBusinessForm />
</div>
`;
